src/azure/Templates/AzureConfigurationTemplate.cshtml

```diff
@@ -12,12 +12,13 @@
 # --------------------------------------------------------------------------
 @Header("# ").TrimMultilineHeader()
 # --------------------------------------------------------------------------
-from msrestazure import AzureConfiguration
+from azure.core.configuration import Configuration, ConnectionConfiguration
+from azure.core.pipeline import policies
 @EmptyLine
 from .version import VERSION
 @EmptyLine
 @EmptyLine
-class @(Model.Name)Configuration(AzureConfiguration):
+class @(Model.Name)Configuration(Configuration):
```

Comment:
There is still a need for Azure configuration,

Reply:
It is azure.core.configuration. We don't use msrestazure.AzureConfiguration any longer.

Reply:
You are in the azure generator here :)